third_party/musl/src/starboard/fcntl/open.c

```diff
@@ -0,0 +1,43 @@
+#if SB_API_VERSION < 16
+
+#include <errno.h>
+#include <fcntl.h>
+#include <sys/stat.h>
+
+#include "starboard/file.h"
+#include "starboard/shared/posix/file_internal.h"
```

Comment:
We can't use Starboard internal implementation outside of Starboard.
I think you might need to emulate file descriptors similar to what Max did for sockets.

Reply:
Removed.